### PR TITLE
Update groovesquid.rb

### PR DESCRIPTION
**Pull request by leoj3n**
Since Grooveshark shut down this app has changed and is now independent

### CODE REVIEW

File: Casks/groovesquid.rb [hunk 1]
```diff
@@ -1,13 +1,12 @@
 cask :v1 => 'groovesquid' do
-  version '0.9.4'
-  sha256 '93ad97b25c5065b60cdceae950c21b12d3802480dafbcb3336f3fd3c6dc77ebe'
+  version :latest
+  sha256 :no_check
 
   # github.com is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/groovesquid.rb]: Comment needs to be removed, as it no longer makes sense.

File: Casks/groovesquid.rb [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
-  url "https://github.com/groovesquid/groovesquid/releases/download/v#{version}/Groovesquid.dmg"
-  appcast 'https://github.com/groovesquid/groovesquid/releases.atom'
+  url 'http://groovesquid.com/download.php?file=Groovesquid.dmg'
   name 'Groovesquid'
-  homepage 'https://groovesquid.com/'
```
Comment (vitorgalvao) [inline Casks/groovesquid.rb]: Trailing slash must remain.

File: Casks/groovesquid.rb [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
-  license :gpl
+  homepage 'https://groovesquid.com'
+  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (vitorgalvao) [inline Casks/groovesquid.rb]: `:gratis`.

**Comment by vitorgalvao** [comment]
Thank you for the fast fixes.

**Comment by Ngrd** [comment]
You are so quick. Thank you! :smile:

**Comment by vitorgalvao** [comment]
Nice catch, @Ngrd. Care to merge with squashed commits ([@leoj3n can’t, right now](https://github.com/caskroom/homebrew-cask/pull/12849#issuecomment-124984500)). I can do it as well, but don’t want to step on your toes, if you’re doing it.

**Comment by Ngrd** [comment]
I guess I have messed up a little bit @vitorgalvao. I need some help here. Sorry for that.

**Comment by vitorgalvao** [comment]
No worries. On it.

**Comment by Ngrd** [comment]
Thank you very much! :cry:

**Comment by vitorgalvao** [comment]
Seems to have gone through, but with another commit and without closing the PR (happens sometimes). No big deal. We can leave it as is.

**Comment by Ngrd** [comment]
Ok, thanks again.

**Pull request closed without merging**